### DISCUSSION TITLE
How can I include videos to the reports?

### DISCUSSION THREAD

**Discussion by a7madgamal** [WasiqB/multiple-cucumber-html-reporter#272] [Q&A] [answered]
I want to include `wdio-video-reporter` videos to the report

**Comment by WasiqB** [comment, marked as answer]
Hi @a7madgamal ,

New version [4.0.0-beta.0](https://github.com/WasiqB/multiple-cucumber-html-reporter/releases/tag/v4.0.0-beta.0) has been released yesterday.

Request you to try the new version and follow the steps described in [the documentation](https://multiple-cucumber-html-reporter.vercel.app/docs/attachments#attach-video) to attach video the report.
